### ISSUE TITLE
VSCode validation Docker Container

### ISSUE THREAD

**Issue by Sakorah** [F5Networks/f5-declarative-onboarding#13]
Hi,

the VSCode validation is not aware about deploying DO via docker container and marks the fields 
```
    "class": "DO",
    "targetHost": "bigip.fqdn.com",
    "targetPort": 443,
    "targetUsername": "admin",
    "targetPassphrase": "admin",
```
as ```Property not allowed```

Greetings,
Sako!

**Comment by seattlevine** [comment]
If you want to validate deploying to a container, you need to point to the remote schema:
"$schema": "https://raw.githubusercontent.com/F5Networks/f5-declarative-onboarding/master/schema/remote.schema.json",

**Comment by Sakorah** [comment]
Hi!

that worked... I only get:
```
	"message": "Property $schema is not allowed.",
```

**Comment by seattlevine** [comment]
I see - yes, that's a bug in the remote schema. Will get it updated for the next release.

Tracking with internal issue 232.

**Comment by Sakorah** [comment]
thanks!

can this be adopted just like in AS3? so just specify the scheme uri with version and not distinguish between remote and base?

**Comment by seattlevine** [comment]
Resolved in 1.5.0
